mods/gen/weapons/other.yaml

```diff
@@ -515,68 +515,54 @@ capture.black_lotus.anim:
 	ValidTargets: Capture
 	TargetActorCenter: true
 	Report: ucapture.aud
-	Projectile: LaserZap
+	Projectile: ArcLaserZap
 		Width: 60
 		Color: 00FF0070
-		SecondaryBeam: true
-		SecondaryBeamWidth: 90
-		SecondaryBeamColor: FF000035
-	Warhead@1Dam: TargetDamage
+		Duration: 20
+	Warhead@1: FlashTarget
+		FlashColor: 00ff0070
+		UsePlayerColor: false
```

Comment:
Why Green? Original game flashed with player color, and the normal capture logic with the AP Infantry also use player color. I don't see the point of making it different here.

![image](https://github.com/MustaphaTR/Generals-Alpha/assets/7933210/29b77782-8f2b-4479-9034-9b7bdb6d6c85)